nixos/modules/services/home-automation/zwave-js.nix

```
@@ -0,0 +1,150 @@
+{config, pkgs, lib, ...}:
+
+with lib;
+
+let
+  cfg = config.services.zwave-js;
+  defaultConfig = {
+  };
+  mergedConfigFile = "/run/zwave-js/config.json";
+  settingsFormat = pkgs.formats.json {};
+in {
+
+  options.services.zwave-js = {
+    enable = mkEnableOption (mdDoc "Start the zwave-js server on boot");
```

This setting is gonna end up weird in the manual. While it's not really obvious, `mkEnableOption` automatically prepends "Whether to enable" before the sentence, effectively making this "Whether to enable Start the zwave-js server on boot". Let's change it up a bit.

```suggestion
    enable = mkEnableOption (mdDoc "the zwave-js server on boot");
```

---

Didn't know that, thanks